mcp/runtime/src/main/java/io/quarkiverse/langchain4j/mcp/runtime/QuarkusMcpToolProvider.java

```diff
@@ -20,9 +21,10 @@
 
 public class QuarkusMcpToolProvider extends McpToolProvider {
 
+    // TODO DefaultMcpResourcesAsToolsPresenter needs to be examined
```

Comment:
We may add an option for the app to provide a custom presenter as a bean - this is not super important for now, but... For now, at the very least, we need a way to turn off the presenter altogether. I guess this should be a config property, like `quarkus.langchain4j.mcp.CLIENT_NAME.expose-resources-as-tools`, and if that is false, then you would pass `null` to the `super` constructor

I see you have some failures in your tests that are caused by exactly this - the presenter adds two new tools that the tests don't expect...